New Checkout Drop-in v1.4.0-beta2 (#273)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/containers/overview.mdx

The `Title` property is a handler used to render a customized title content.

:::note[Checkout containers]

> In the next pages, we will see some containers which are able to customize the `Title` and we will describe how to use it merging props.

Reviewer: ```suggestion
In the pages that follow, you will learn how to customize the `Title` and how to use it.
```
